app/controllers/memberships_controller.rb

```diff
@@ -13,10 +13,14 @@ def show
   # GET /memberships/new
   def new
     @membership = Membership.new
+    @groups = Group.all.where(user_id: current_user)
+    @contacts = Contact.all.where(user_id: current_user).order(:last_name)
```

Comment:
I tend to avoid using `where` unless I *really* have to.

```suggestion
    @groups = current_user.groups
    @contacts = current_user.contacts.order(:last_name)
```